Bottom: 28ddc198e92a31c56fa47afdcff47111e6b659ae
Top:    42888768b226550ec9f938e6eb7c283f50d4a71d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 00:49:36 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index b2add35..ff94ccf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
